db/migrate/20210330193831_change_column_name.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeColumnName < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :course_requests, :course_name, :course_name_full
+  end
+  def up
+    remove_column :course_requests, :request_id
```

Comment:
Why is this up?

Reply:
it's from an old migration when I was making changes to course_request, and I ended up removing the request_id since there's a built-in id